captum/optim/_param/image/transform.py

```diff
@@ -135,14 +135,22 @@ class CenterCrop(torch.nn.Module):
         pixels_from_edges (bool, optional): Whether to treat crop size
             values as the number of pixels from the tensor's edge, or an
             exact shape in the center.
+        offset_left (bool, optional): If the cropped away sides are not
+            equal in size, offset to the left. Default is set to False
```

Comment:
nit: offset to the left means that the +1 will be on the right side ? It would be good to describe it here on an example.

Reply:
Okay I changed the description to:

```
offset_left (bool, optional): If the cropped away sides are not
    equal in size, offset center by +1 to the left and/or top.
    Default is set to False. This parameter is only valid when
    pixels_from_edges is False.
```

Reply:
Thank you!